src/core/bluetoothreceiver.cpp

```diff
@@ -59,7 +84,8 @@ void BluetoothReceiver::connectDevice( const QString &address )
 
 void BluetoothReceiver::stateChanged( const QgsGpsInformation &info )
 {
-  mLastGnssPositionInformation = GnssPositionInformation( info.latitude, info.longitude, info.elevation, info.speed, info.direction, info.satellitesInView, info.pdop,
+  // QgsGpsInformation's speed is served in km/h, translate to m/h
```

Comment:
That comment (and the related fix) was correct: according to the doc (https://qgis.org/api/classQgsGpsInformation.html#a1deea391d08706b0c7b8f6df2993247c), QgsGpsInformation::speed represents the ground speed, in km/h.

Reply:
Shouldn't it multiply then instead of divide with 1000?
And m/h is a weird format. Should it be m/s? Like here https://doc.qt.io/qt-5/qml-qtpositioning-position.html#speed-prop

Reply:
Yes I was trying to align it to that, and you're right re multiplication, will adjust.